gender_novels/analysis/dependency_parsing.py

```diff
@@ -0,0 +1,41 @@
+import urllib
+from nltk.parse.stanford import StanfordDependencyParser
+from gender_novels.corpus import Corpus
+from nltk.tokenize import sent_tokenize
+
+if __name__ == "__main__":
+
+    path_to_jar = "assets/stanford-parser.jar"
+    path_to_models_jar = "assets/stanford-parser-3.9.1-models.jar"
+
+    # The jar files are too big to commit directly, so download them
+    url_to_jar = "http://www.trecento.com/dh_lab/nltk_jar/stanford-parser.jar"
+    url_to_models_jar = "http://www.trecento.com/dh_lab/nltk_jar/stanford-parser-3.9.1-models.jar"
+    urllib.request.urlretrieve(url_to_jar, path_to_jar)
+    urllib.request.urlretrieve(url_to_models_jar, path_to_models_jar)
+
+    dependency_parser = StanfordDependencyParser(path_to_jar, path_to_models_jar)
+
+    novels = Corpus('sample_novels').novels
+
+    novel = novels[0]
+    sentences = sent_tokenize(novel.text)
```

Comment:
If running out of memory -- use one of your teammate's split sentence routines and just tokenize one sentence at a time.  This should be a function to figure out something for one sentence.  Break down the work into the smallest possible functions, writing long scripts to do all at once is 1980s-style programming, so I'm the only one in the lab old enough to be forgiven for doing this.  :-)

Reply:
Will do! I've removed the code handling novels for now, but I'll do this when I get to this point.